rotkehlchen/tests/test_users.py

```diff
@@ -104,7 +105,7 @@ def test_users_query_permission_error(
     os.mkdir(not_allowed_dir)
     os.chmod(not_allowed_dir, 0o200)
     os.mkdir(allowed_user_dir)
-    Path(Path(allowed_user_dir) / 'rotkehlchen.db').touch()
+    Path(Path(allowed_user_dir) / USERDB_NAME).touch()
```

Comment:
this second path is not need... is it?

```suggestion
    (Path(allowed_user_dir) / USERDB_NAME).touch()
```

Reply:
actuallly none of them are!